client/components/promo-section/promo-card/cta.tsx

```diff
@@ -114,7 +115,7 @@ const PromoCardCta: FunctionComponent< Props & ConnectedProps > = ( {
 			{ ctaBtn }
 			{ learnMore && (
 				<Button borderless className="promo-card__cta-learn-more" { ...learnMore }>
-					{ translate( 'Learn more' ) }
+					{ learnMoreLink && learnMoreLink.label ? learnMoreLink.label : translate( 'Learn More' ) }
```

Comment:
```suggestion
					{ learnMoreLink?.label || translate( 'Learn more' ) }
```

Two things:

- Let's keep "Learn more" with the same capitalization (lowercase `m`) to avoid risking breaking existing translations.
- We can just go with the `?.` optional chaining here.
  We are already sure that `learnMoreLink` exists because `learnMore` is valued, so we only really need to check the `label` property.